examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -1,25 +1,55 @@
-describe('basic test', () => {
+describe('Basic search', () => {
   beforeEach(() => {
-    cy.server();
-    cy.fixture('basic/query.json').as('queryJSON');
-    cy.route('POST', '**/query?version=2019-01-01', '@queryJSON').as('postQuery');
+    cy.server({ force404: true });
```

Comment:
do we want to move this into the cypress configuration setup? i think it would be valuable across all specs